### DISCUSSION TITLE
训练仅识别数字的模型（自定义了字典，自己打标的数据和标签），但识别出的结果中文。

### DISCUSSION THREAD

**Discussion by TheodorePTP** [PaddlePaddle/PaddleOCR#12654] [Q&A] [answered]
训练仅识别数字的模型（自定义了字典（字典内容为手动输入的0-9），自己用paddlelabel工具打标的数据和标签），但识别出的结果为中文。

尝试重新修改配置了字典，但没有作用。
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/106580844/7d29051c-1850-4ac6-9087-7e19ef4adebc)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/106580844/b306aa77-1945-481e-8b79-755f44e27258)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/106580844/f9d8c29a-7cfd-4f59-bc48-37c627331278)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/106580844/9b67e8ab-c373-442b-8c1c-e8f34bd94ab1)

**Comment by zzgHiker** [comment, marked as answer]
可能是推理时的字符字典没有指定的原因，参数：--rec_char_dict_path

**Comment by TheodorePTP** [comment]
> 可能是推理时的字符字典没有指定的原因，参数：--rec_char_dict_path
已解决，感谢
